examples/config/broker/runtime.properties

```diff
@@ -15,8 +15,8 @@
 # limitations under the License.
 #
 
-druid.host=localhost
-druid.port=8080
+#druid.host=localhost
```

Comment:
It is unclear if these are commented out because they don't work or because they are defaults. Can they just be removed?